doc/source/examples/metrology/Creating Network Definitions for SOLT Calibration Standards.ipynb

```diff
@@ -0,0 +1,1004 @@
+    "in the [*Preparation*](#preparation) section).\n",
+    "\n",
+    "\n",
+    "## Example: HP-Agilent-Keysight Coefficient Format\n",
+    "\n",
+    "After the necessary background is introduced, let's begin.\n",
+    "\n",
+    "For the purpose of this guide, we're going to model the Keysight 85033E,\n",
+    "3.5 mm, 50 Ω, DC to 9 GHz calibration kit, with the following coefficients\n",
+    "[[4](#ref4)].\n",
+    "\n",
+    "|   Parameter  |            Unit             |    Open   |   Short   |  Load   |   Thru   |\n",
+    "| ------------ | --------------------------- | --------- | --------- | ------- | -------- |\n",
+    "| $\\text{C}_0$ |  $10^{−15} \\text{ F}$       |   49.43   |           |         |          |\n",
+    "| $\\text{C}_1$ |  $10^{−27} \\text{ F/Hz}$    | -310.1    |           |         |          |\n",
+    "| $\\text{C}_2$ |  $10^{−36} \\text{ F/Hz}^2$  |   23.17   |           |         |          |\n",
+    "| $\\text{C}_3$ |  $10^{−45} \\text{ F/Hz}^3$  |   -0.1597 |           |         |          |\n",
+    "| $\\text{L}_0$ |  $10^{−12} \\text{ H}$       |           |    2.077  |         |          |\n",
+    "| $\\text{L}_1$ |  $10^{−24} \\text{ H/Hz}$    |           | -108.5    |         |          |\n",
+    "| $\\text{L}_2$ |  $10^{−33} \\text{ H/Hz}^2$  |           |    2.171  |         |          |\n",
+    "| $\\text{L}_3$ |  $10^{−42} \\text{ H/Hz}^3$  |           |   -0.01   |         |          |\n",
+    "|  Resistance  |         $\\Omega$            |           |           |    50   |          |\n",
+    "| Offset $Z_0$ |         $\\Omega$            |    50     |     50    |    50   |    50    |\n",
+    "| Offset Delay |            ps               |  29.242   |   31.785  |     0   |     0    |\n",
+    "| Offset Loss  | $\\text{G}\\Omega$ / s        |   2.2     |    2.36   |     2.3 |     2.3  |"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Circuit Model\n",
+    "\n",
+    "Before we start creating their network definitions, we first need to know\n",
+    "the underlying circuit model and the meaning of these coefficients.\n",
+    "\n",
+    "![Circuit Schematic](calkit-schematic.svg)\n",
+    "\n",
+    "As this schematic shows, this is the HP-Agilent-Keysight model for\n",
+    "a calibration standard.\n",
+    "\n",
+    "The first part is an \"offset\" lossy transmission\n",
+    "line, defined using three parameters: (1) a real characteristic impedance\n",
+    "of a lossless line - usually the system impedance, but for waveguide\n",
+    "standard, a special normalized value of `1` is used, (2) a delay -\n",
+    "represents its electrical length, given in picoseconds, and (3) a loss.\n",
+    "The loss is given in a somewhat unusual unit - gigaohms per second. With\n",
+    "the three parameters, one can calculate the *propagation constant*\n",
+    "($\\gamma$) and the complex *characteristic impedance* ($Z_c$) of the\n",
```

Comment:
I understand that Z_c is distinct from Z_0, in particular because of the discussion below. So I would have added a note saying the difference in notation is intended, such as:
```suggestion
    "($\\gamma$) and the complex *characteristic impedance* ($Z_c$) of the (note that $Z_c$ may be different from the port impedance $Z_0$)\n",
```

Reply:
I added an slightly longer explanation in the next paragraph.